client/tests/unit/specs/testPath.spec.ts

```diff
@@ -14,4 +14,13 @@ describe('Path', () => {
     expect(() => parse('')).to.throw('Path should be at least "/" and start with "/"');
     expect(() => parse('a/b')).to.throw('Path should be at least "/" and start with "/"');
   });
+
+  it('test path joining', async () => {
+    expect(join('/', 'a')).to.equal('/a');
+    expect(join('', '/')).to.equal('/');
+    expect(join('', '/a')).to.equal('/a');
+    expect(join('/a', 'b')).to.equal('/a/b');
+    expect(join('/a/', '/b')).to.equal('/a/b');
```

Comment:
same line as 24, should probably be:
```suggestion
    expect(join('/a', '/b')).to.equal('/a/b');
```